testsuite/tests/backtrace/backtrace3.ml

```diff
@@ -21,7 +21,24 @@ let g msg =
   | exception (Error "c") ->
       (* according to the current re-raise policy (a static condition),
          this does not re-raise *)
-      raise (Error "c")
+      print_string "c"; print_newline(); raise (Error "c")
+  | exception (Error "d" as exn as exn2) ->
```

Comment:
I think the test still makes sense if this is `_exn2`, which should eliminate Warning 26

Reply:
That sounds right yes, I'll fix that.